cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -364,6 +364,9 @@ foreach(_VERSION_ENTRY ${TOOLCHAIN_VERSIONS_TXT})
   set(${_VARIABLE_NAME} ${_VARIABLE_VALUE})
 endforeach()
 
+set(THIRDPARTY_MIRROR_URL
+    "https://github.com/ursa-labs/thirdparty/releases/download/apache-arrow-7.0.0")
```

Comment:
Can we use https://apache.jfrog.io/artifactory/arrow/thirdparty here?

Reply:
We can, but the boost tarball there is the previous trimmed version.

I was thinking to keep the github mirror so we can see how reliable jfrog is after the release.

Reply:
OK. It makes sense.

Reply:
Though I've just set up the jfrog directory, so we can just simply switch to that.